src/pages/admin/quizzes/QuizEdit/index.js

```diff
@@ -48,7 +107,61 @@ const QuizEdit = () => {
             <Button className={style.sidebarButtons}>Add a Question</Button>
             <Button className={style.sidebarButtons}>Change Category</Button>
           </div>
-          {/*************** INSERT <QUESTION FORM> CODE HERE ***************/}
+          <div>{question_type(questionType)}</div>
+          <div className={style.formGap}>
+            <Dropdown onSelect={question}>
+              <Form>
+                <Form.Label className={style.inputTitle}>
+                  Question Type
+                </Form.Label>
+              </Form>
+              <Dropdown.Toggle
+                variant="link"
+                id="dropdown-basic"
+                bsPrefix="none"
+                className={style.dropdownStyle}
+              >
+                {choice(questionType)}
+                <RiArrowDropDownLine className={style.iconSize} />
+              </Dropdown.Toggle>
+              <Dropdown.Menu className={style.dropdownMenuStyle}>
+                {types.map((type, idx) => {
+                  return (
+                    <Dropdown.Item key={idx} eventKey={type.value}>
+                      {type.name}
+                    </Dropdown.Item>
+                  );
+                })}
+              </Dropdown.Menu>
+            </Dropdown>
+            <div className={style.formSpacing}>
+              <Dropdown onSelect={time}>
+                <Form>
+                  <Form.Label className={style.inputTitle}>
+                    Time Limit
+                  </Form.Label>
+                </Form>
+                <Dropdown.Toggle
+                  variant="link"
+                  id="dropdown-basic"
+                  bsPrefix="none"
+                  className={style.dropdownStyle}
+                >
+                  {timeLimit} seconds
+                  <RiArrowDropDownLine className={style.iconSize} />
+                </Dropdown.Toggle>
+                <Dropdown.Menu className={style.dropdownMenuStyle}>
+                  {limit.map((time, idx) => {
+                    return (
+                      <Dropdown.Item key={idx} eventKey={time}>
+                        {time} seconds
+                      </Dropdown.Item>
+                    );
+                  })}
+                </Dropdown.Menu>
+              </Dropdown>
+            </div>
+          </div>
```

Comment:
please create another component file and put the contents there to avoid bloating this file